src/renderers/WebGLRenderer.js

```diff
@@ -1416,7 +1416,7 @@ class WebGLRenderer {
 
 				_transmissionRenderTarget = new WebGLRenderTarget( 1, 1, {
 					generateMipmaps: true,
-					type: extensions.has( 'EXT_color_buffer_half_float' ) ? HalfFloatType : UnsignedByteType,
+					type: ( extensions.has( 'EXT_color_buffer_half_float' ) || extensions.has( 'EXT_color_buffer_float' ) ) ? HalfFloatType : UnsignedByteType,
```

Comment:
AFAIK, there is no system that supports `EXT_color_buffer_float ` but not `EXT_color_buffer_half_float`. Since the transmission render target only uses `HalfFloatType` and not `FloatType`, I'm afraid this addition does not make sense.

Reply:
Confusingly the `EXT_color_buffer_float` extension may imply that half float types are supported. From the [spec](https://registry.khronos.org/webgl/extensions/EXT_color_buffer_float/):

> The following floating-point internal formats become color-renderable: R16F, RG16F, RGBA16F, R32F, RG32F, RGBA32F and R11F_G11F_B10F. A renderbuffer or a texture with a color-renderable internal format can be used as a rendering target by attaching it to a framebuffer object as a color attachment.

If a device reports that `EXT_color_buffer_float` but for some reason doesn't report `EXT_color_buffer_half_float` we should still support rendering to half float buffers.

Reply:
Then let's merge just for the case a device only reports `EXT_color_buffer_float` support.